kong/plugins/pre-function/_handler.lua

```diff
@@ -8,7 +9,22 @@ local config_cache do
 
   local no_op = function() end
 
-  local sandbox_opts = { env = { kong = kong, ngx = ngx } }
+  local shm_name = "kong_db_cache"
+  local cache_name = "serverless_" .. shm_name
+  local cache = resty_mlcache.new(cache_name, shm_name, { lru_size = 1e4 })
+  local sandbox_kong = setmetatable({
+    cache = cache
+  }, {
+    __index = function(self, k)
+      if k == "configuration" then
+        self.configuration = kong.configuration.remove_sensitive()
+        return self.configuration
+      end
+      return kong[k]
+    end
```

Comment:
It seems like this could be simplified further:
```suggestion
  local sandbox_kong = setmetatable({
    cache = cache,
    configuration = kong.configuration.remove_sensitive(),
  }, { __index = kong }
    end
```